R/stpca.init.R

```diff
@@ -0,0 +1,88 @@
+#' Initialises an stpca object.
+#'
+#' @param X Data
+#' @param k Latent dimensionality
+#' @param locations the coordinates of each dimenion in X
+#' @param covar.fn covariance function to generate K_beta
+#' @param covar.fn.d gradient of the covariance function with respect to hyperparameters
+#' @param beta0 initial hyperparameters
+#' @param trace amount of reporting. 0=none, 1=low, 2=high
+#' @param max.dist Maximum distance between features to consider
+#' @param constraints constraints on beta, of the form used in maxLik
```

Comment:
If you want to jazz up your documentation (and who doesn't?) you could use \link[maxLik]{maxLik} to link to the maxLik documentation.